test/protobuf/protoc/generator/message_test.exs

```diff
@@ -7,19 +7,19 @@ defmodule Protobuf.Protoc.Generator.MessageTest do
   alias Protobuf.TestHelpers
 
   test "generate/2 has right name" do
-    ctx = %Context{}
+    ctx = %Context{package: "pkg.name"}
     desc = %Google.Protobuf.DescriptorProto{name: "Foo"}
     {[], [{_mod, msg}]} = Generator.generate(ctx, desc)
-    assert msg =~ "defmodule Foo do\n"
-    assert msg =~ "use Protobuf, protoc_gen_elixir_version: \"#{Util.version()}\"\n"
+    assert msg =~ "defmodule Pkg.Name.Foo do\n"
+    assert msg =~ "use Protobuf, full_name: \"pkg.name.Foo\", protoc_gen_elixir_version: \"#{Util.version()}\"\n"
```

Comment:
Isn't it better to include a function in the template? Since the function must exist anyway (and is currently generated in the use-macro)

Reply:
I am unsure, either way is fine by me, I need to make it work first 😭

Reply:
About this, I am following the existing code style, being an argument is so that I can enforce the callback inside, similar to the rest of the code